### ISSUE TITLE
Strings using single quotes not parsed correctly.

### ISSUE THREAD

**Issue by november-yankee** [nilslice/protolock#65]
I only have a description of the issue.

Someone reported a `.proto` file with `reserved 'is_success', 'error_response'`. The `proto.lock` file that's generated has:

```
                        "reserved_names": [
                            "",
                            "",
                            "",
                            ""
                        ]
```

If double quotes are used in the `.proto` file, the generated file looks better.

**Comment by nilslice** [comment]
Do you know if single-quoted reserved field names are valid syntax? 

It is a bug to fill `proto.lock` with empty reserved names, and I have a fix to PR upstream to the proto parser lib that protolock depends on. However, that parser doesn't handle or recognize these single-quoted reserved names at all.

Can you verify that those files successfully compile?

**Comment by november-yankee** [comment]
Hmm, according to that grammar, there shouldn't be any quotes unless I'm missing something.

**Comment by nilslice** [comment]
@november-yankee - sorry, you beat me to the punch while updating..

> I ran some tests on protos using this single-quote syntax, and protoc had no issues with it.. the docs are a bit hard to interpret :)

> So, it does appear valid, and since protoc has no issues, this does appear to be a bug. I will look into the proto parser code to see if I can see where this originates.

> Thanks for reporting!

**Comment by nilslice** [comment]
@november-yankee I have opened up an issue with the parser lib and will keep you updated. Feel free to follow along here: https://github.com/emicklei/proto/issues/94

**Comment by nilslice** [comment]
Updates have moved to this PR, https://github.com/emicklei/proto/pull/96.

**Comment by nilslice** [comment]
Fixed in #66. Thanks for reporting, @november-yankee!
